test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs

```diff
@@ -1154,9 +1155,11 @@ public void ServiceBus_SDKTypeBindings()
 
             Assert.Equal(2, functions.Count());
 
+            var extensionReference = typeof(ServiceBusExtensionStartup).Assembly.GetCustomAttribute<ExtensionInformationAttribute>();
+
             AssertDictionary(extensions, new Dictionary<string, string>
             {
-                { "Microsoft.Azure.WebJobs.Extensions.ServiceBus", "5.16.0" },
+                { "Microsoft.Azure.WebJobs.Extensions.ServiceBus", extensionReference.ExtensionVersion },
```

Comment:
@satvu / @aishwaryabh  got tired of always missing this test when bumping the reference, so updating this to use the referenced version.